x-pack/plugins/observability/public/pages/alerts/translations.ts

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
:+1: I like the idea of moving translation constants to a separate file. Not sure about committing to adopting it as a convention just yet. There's some advantages to keeping them all in-line in the file: sometimes we debug by grepping for certain text strings and it's nice to be taken right to the code, and there are cases where we have to use `<FormattedMessage>` that might be awkward to import from a separate module. But let's do it here and see how maintainability works out.